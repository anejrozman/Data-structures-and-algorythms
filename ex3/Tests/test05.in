14,41,5
2.8358954749413128,7.119754798205999,9.0304731139107,2.9776314718846617,5.358330398370643,1.8583550249959813,8.77546690003287,2.1094677045489076,2.032320199784325,4.774870650417539,2.923673042040661,4.181597881403015,6.699540622892184,5.287324742705607
4,2,640.163
7,1,514.283
5,1,517.71
13,2,262.137
8,10,775.514
0,1,7.1
6,8,648.412
4,10,266.723
12,9,399.29
2,8,147.489
7,3,277.611
6,1,664.364
3,13,154.255
6,7,673.73
8,12,38.832
8,1,166.811
2,5,39.219
6,4,40.657
9,3,171.715
11,3,688.164
13,10,818.619
11,0,310.675
2,0,950.968
8,13,439.04
0,4,186.209
9,2,61.67
5,13,574.404
7,8,326.312
6,13,593.189
11,5,851.977
6,12,474.004
9,4,857.326
5,8,860.859
2,11,220.357
4,12,91.408
6,0,857.278
10,9,357.979
8,0,615.541
12,10,371.572
7,9,628.727
12,13,891.022
