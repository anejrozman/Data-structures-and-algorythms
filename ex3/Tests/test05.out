9,1014.7628
2,15.7437
1,0
1,0
1,0
